pkg/gadgets/seccomp/tracer/tracer.go

```diff
@@ -34,6 +34,12 @@ const (
 type Tracer struct {
 	collection *ebpf.Collection
 	seccompMap *ebpf.Map
+
+	// progLink links the BPF program to the tracepoint.
+	// A reference is kept so it can be closed it explicitely, otherwise
```

Comment:
```suggestion
	// A reference is kept so it can be closed it explicitly, otherwise
```

You may want to add a linter that checks this (runc has one) in the future :)